open-api/rest-catalog-open-api.yaml

```diff
@@ -1435,12 +1463,66 @@ components:
           additionalProperties:
             type: string
 
+    TokenRequest:
+      description:
+        OAuth2 client credentials or token exchange request
+
+
+        See -
+
+        - Client credentials flow - https://datatracker.ietf.org/doc/html/rfc6749#section-4.4
+
+        - Token exchange flow - https://datatracker.ietf.org/doc/html/rfc8693
+      type: object
+      required:
+        - grant_type
+      properties:
+        grant_type:
+          type: string
+          enum:
+            - client_credentials
+            - urn:ietf:params:oauth:grant-type:token-exchange
```

Comment:
@bryanck, do you know if the `urn:ietf:..` prefix can be used with `client_credentials` as well? If so, then I'd probably change so these are uniform.

Reply:
Not that I'm aware of, what you have here looks correct to me, per the OAuth2 specs.